doc/man7/EVP_MD-SHA2.pod

```diff
@@ -0,0 +1,50 @@
+=pod
+
+=head1 NAME
+
+EVP_MD-SHA2 - The SHA2 EVP_MD implementation
+
+=head1 DESCRIPTION
+
+Support for computing SHA2 digests through the B<EVP_MD> API.
+
+=head2 Identity
+
+This implementation is identified with this name and properties, to be
+used with EVP_MD_fetch():
+
+=over 4
+
+=item "SHA2-224", "provider=default" or "provider=fips"
+
+=item "SHA2-256", "provider=default" or "provider=fips"
+
+=item "SHA2-384", "provider=default" or "provider=fips"
+
+=item "SHA2-512", "provider=default" or "provider=fips"
+
+=item "SHA2-512/224", "provider=default"
+
+=item "SHA2-512/256", "provider=default"
```

Comment:
Should we document aliases or not?

Reply:
I was wondering the same thing. I can certainly mention them all, no problem.

Reply:
Done